vunit/test/unit/test_ui.py

```diff
@@ -255,6 +255,46 @@ def test_add_source_files(self):
         for file_name in files:
             lib.get_source_file(file_name).name.endswith(file_name)
 
+    def test_add_source_files_from_csv(self):
+        csv = """
+        lib,  tb_example.vhdl
+        lib1 , tb_example1.vhd
+        lib2, tb_example2.vhd
+        """
+
+        libraries = ['lib', 'lib1', 'lib2']
+        files = ['tb_example.vhdl', 'tb_example1.vhd', 'tb_example2.vhd', ' tb,ex3.vhd']
+
+        self.create_csv_file('test_csv.csv', csv)
+        for file_name in files:
+            self.create_file(file_name)
+
+        ui = self._create_ui()
+        ui.add_source_files_from_csv('test_csv.csv')
+
+        for index, library_name in enumerate(libraries):
+            file_name = files[index]
+            file_name_from_ui = ui.add_source_files_from_csv(file_name, library_name)
```

Comment:
This line does not make sense. You add the file_name as a csv file but the file is an empty vhdl_file.
Did you mean to do get_source_file?